### PR TITLE
Add license file

### LINKED ISSUE #2: License is missing
Update LICENSE file and each file's license.

### PR DESCRIPTION
**Pull request by sangwoo108**
fix #2 #3

### CODE REVIEW

**Comment by simonhong** [comment]
I think @fmarier should check this PR :)

File: index.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (fmarier) [inline index.js]: We normally add a copyright statement with the date to the top of the notice:
```
/* Copyright (c) 2022 The Brave Authors. All rights reserved.
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at https://mozilla.org/MPL/2.0/. */
```

**Review by fmarier — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (sangwoo108) [reply index.js]: That's what I was curious! Sometimes I ran into copyrights without the year. Thank you for confirmation. Done.

Comment (fmarier) [reply index.js]: Indeed, there are a lot of old / incorrect ones still around in the codebase and so it's easy to forget to add it if copy/pasting from one of those files.

**Review by fmarier — APPROVED** [review]

**Comment by sangwoo108** [comment]
Thank you for reviewing this!

**Pull request merged**